_includes/v20.1/misc/external-urls.md

```diff
@@ -33,18 +29,18 @@ If your environment requires an HTTP or HTTPS proxy server for outgoing connecti
 
 - <sup>4</sup> The file system backup location on the NFS drive is relative to the path specified by the `--external-io-dir` flag set while [starting the node](cockroach-start.html). If the flag is set to `disabled`, then imports from local directories and NFS drives are disabled.
 
-- <sup>5</sup>  The host component of NFS/Local can either be empty or the `nodeID`. If the `nodeID` is specified, it is currently ignored (i.e., any node can be sent work and it will look in its local input/output directory); however, the `nodeID` will likely be required in the future.
+- <sup>5</sup>  The host component of NFS/Local can either be empty or the `nodeID`. If you do not specify a `nodeID` when using `nodelocal` storage, the individual data files will be in the `extern` directories of arbitrary nodes and will likely not work as intended. To work correctly, each node must have the [`--external-io-dir` flag](cockroach-start.html#general) point to the same NFS mount or other network-backed, shared storage. <span class="version-tag">New in v20.1:</span> If a `nodeID` is provided, the data files will be in the `extern` directory of the specified node. The `nodeID` will likely be required in the future.
```

Comment:
The last sentence is confusing to me: "`nodeID` will likely be required in the future". Does it mean that `nodeID` is optional right now and might be mandatory in the future and so the users should use it now to future-proof their setup? Or will it be required for something else entirely?